server.py

```diff
@@ -12,6 +12,7 @@
 
 IDENTITY = 'voice_test'
 CALLER_ID = 'quick_start'
+CALLER_PHONE = '+1****'
```

Comment:
Yeah. The quickstart guide will need a section about obtaining a twilio number and replacing this value.